pkg/cluster/kafkaservice/kafkaservice.go

```diff
@@ -0,0 +1,65 @@
+package kafkaservice
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/redhat-developer/app-services-cli/pkg/api/kas"
+	"github.com/redhat-developer/app-services-cli/pkg/cluster"
+	"github.com/redhat-developer/app-services-cli/pkg/cluster/constants/kafka"
+	"github.com/redhat-developer/app-services-cli/pkg/cluster/utils"
+	"github.com/redhat-developer/app-services-cli/pkg/kafka/kafkaerr"
+)
+
+type KafkaConnection struct {
```

Comment:
Missing comments. Also this will clash with actual Connection object in constants. Let's try to avoid using the same names across the board. Specially that this is not DTO/Entity but functional/service class